test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -52,23 +52,30 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+type operationCode string
+
 const (
-	configFile               = "config/performance-config.yaml"
-	createNodesOpcode        = "createNodes"
-	createNamespacesOpcode   = "createNamespaces"
-	createPodsOpcode         = "createPods"
-	createPodSetsOpcode      = "createPodSets"
-	churnOpcode              = "churn"
-	barrierOpcode            = "barrier"
-	sleepOpcode              = "sleep"
-	extensionPointsLabelName = "extension_point"
+	createNodesOpcode      operationCode = "createNodes"
+	createNamespacesOpcode operationCode = "createNamespaces"
+	createPodsOpcode       operationCode = "createPods"
+	createPodSetsOpcode    operationCode = "createPodSets"
+	churnOpcode            operationCode = "churn"
+	barrierOpcode          operationCode = "barrier"
+	sleepOpcode            operationCode = "sleep"
+)
 
+const (
```

Comment:
just leaving empty lines within the `const` block should be enough, but up to you.

Reply:
Block can help to declare a group of related constants or variables, prefer to keep it.